pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl

```diff
@@ -114,6 +169,50 @@ workflow RNAWithUMIsPipeline {
 	}
 }
 
+task FastqToUbam {
+    input {
+        File r1_fastq
+        File r2_fastq
+        String output_basename
+        String library_name
+        String platform
+        String platform_unit
+        String read_group_name
+        String sequencing_center
+
+        Int disk = ceil(size(r2_fastq, "GiB") * 2.2)
```

Comment:
maybe sum r1_fastq and r2_fastq here?  I would assume they are almost the same size, but can't be sure.